aptos-move/framework/move-stdlib/sources/bit_vector.move

```diff
@@ -145,95 +173,128 @@ module std::bit_vector {
     /// Returns the length of the longest sequence of set bits starting at (and
     /// including) `start_index` in the `bitvector`. If there is no such
     /// sequence, then `0` is returned.
-    public fun longest_set_sequence_starting_at(bitvector: &BitVector, start_index: u64): u64 {
-        assert!(start_index < bitvector.length, EINDEX);
+    public fun longest_set_sequence_starting_at(
+        bitvector: &BitVector,
+        start_index: u64
+    ): u64 {
+        assert!(
+            start_index < bitvector.length,
+            EINDEX
+        );
         let index = start_index;
 
         // Find the greatest index in the vector such that all indices less than it are set.
-        while ({
-            spec {
-                invariant index >= start_index;
-                invariant index == start_index || is_index_set(bitvector, index - 1);
-                invariant index == start_index || index - 1 < vector::length(bitvector.bit_field);
-                invariant forall j in start_index..index: is_index_set(bitvector, j);
-                invariant forall j in start_index..index: j < vector::length(bitvector.bit_field);
-            };
-            index < bitvector.length
-        }) {
+        while (
+            {
+                spec {
+                    invariant index >= start_index;
+                    invariant index == start_index || is_index_set(bitvector, index -
+                            1);
```

Comment:
This line break here looks strange to me.

Reply:
fixed